starters/next13-zustand-bulma/src/app/api-example/page.tsx

```diff
@@ -19,7 +32,10 @@ export default function ApiExamplePage() {
         >
           API Example: Fetch Data using API route
         </h1>
-        <div className="block px-5">{/* Add Greeting Component Here */}</div>
+        <div className="block px-5">
+          {/* @ts-expect-error Async Server Component - please read: https://beta.nextjs.org/docs/data-fetching/fetching#asyncawait-in-server-components for more info */}
+          <Greeting greeting={greeting}/>
```

Comment:
We should also probably wrap this in a `<Suspense>` component with a fallback, like a [spinner](https://bulma.io/documentation/utilities/mixins/#loader) or something


https://user-images.githubusercontent.com/76821/227620647-a8b0e2ad-e327-4f7f-a2f5-aab7e3fc35cb.mov